app/src/components/Partners/ResearchSection.jsx

```diff
@@ -65,6 +68,15 @@ class ResearchSection extends Component {
               <img src={mccauleyLogo} alt="mccauley logo" className={ResearchSectionStyle['image-slider']}></img>
             </div>
             <div>
+              <img src={bmsLogo} alt="bms logo" className={ResearchSectionStyle['image-slider']}></img>
```

Comment:
can you make sure the order matches the "Research Program" page, please? alfabetical order. thanks